src/pages/ZNS/ZNS.tsx

```diff
@@ -489,22 +488,18 @@ const ZNS: React.FC<ZNSProps> = ({ domain, version, isNftView: nftView }) => {
 				style={{
 					opacity: hasLoaded ? 1 : 0,
 					transition: 'opacity 0.2s ease-in-out',
-					paddingTop: 145,
+					paddingTop: 271,
 				}}
 			>
-				{/* Nav Bar */}
-				{/* TODO: Make a more generic Nav component and nest FilterBar and TitleBar */}
-				<FilterBar
+				<PageHeader
```

Comment:
My thinking behind having a PageHeader container here was that the ZNS page file could over time be refactored to look something like:

```
<PageHeader>
    <NavBar />
    <Banner />
</PageHeader>
<PageContent>
    <SomeContent />
    <SomeContent />
</PageContent>
```